portal-web/test/functional/com/liferay/portalweb/tests/MyDMTests/MyDMTests.testcase

```diff
@@ -0,0 +1,145 @@
+@component-name = "portal-document-management"
+definition {
+
+	property testray.main.component.name = "Document Management";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
+
+		HeadlessSite.addSite(siteName = "Test Site Name");
+
+		JSONLayout.addPublicLayout(
+			groupName = "Test Site Name",
+			layoutName = "Documents and Media Page");
+
+		JSONLayout.addWidgetToPublicLayout(
+			column = 1,
+			groupName = "Test Site Name",
+			layoutName = "Documents and Media Page",
+			widgetName = "Documents and Media");
+
+		Navigator.gotoSitePage(
+			pageName = "Documents and Media Page",
+			siteName = "Test Site Name");
+
+		ProductMenu.gotoPortlet(
+			category = "Content & Data",
+			portlet = "Documents and Media");
+	}
+
+	tearDown {
+		PortalInstances.tearDownCP();
+
+		JSONGroup.deleteGroupByName(groupName = "Test Site Name");
+	}
+
+	@description = "Can Create a Documents and Media"
+	@priority = 5
+	test CanCreateDM {
+		task ("When the user create a Documents and Media (Upload the file)") {
+			DMDocument.addCP(
+				dmDocumentFile = "Document_3.txt",
+				dmDocumentTitle = "Document and Media");
+
+			Alert.viewSuccessMessageText(successMessage = "Your request completed successfully.");
+		}
+
+		task ("Then the Documents and Media should display on the page") {
+			Navigator.gotoSitePage(
+				pageName = "Documents and Media Page",
+				siteName = "Test Site Name");
+
+			DMDocument.viewPG(dmDocumentTitle = "Document and Media");
+		}
+	}
+
+	@description = "Can Delete DM"
+	@priority = 5
+	test CanDeleteDM {
+		task ("And create 2 Documents and Media (Upload the file)") {
+			DMDocument.addCP(
```

Comment:
this creating can be made by a json macro, as creating a DM is not the main goal of this test